lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2876,9 +2896,14 @@ class ConvertInterpolateOp
                 [&](OpBuilder &b, Location loc, ValueRange args) {
                   Value retVal;
                   if (mode.substr(0, 7) == "nearest") {
+                    std::string coordTfMode = mode.substr(7, mode.find(","));
```

Comment:
I think the second argument to substr should be the length of the substring rather than the end position. This might need to be something like `mode.find(",") - 7`.